packages/cli/test/commands/start.test.ts

```diff
@@ -0,0 +1,36 @@
+let onSpy: jest.Mock;
+const spawn = jest.fn(() => {
+  onSpy = jest.fn(function on(_: string, callback: (_:number) => {}) {
+    callback(0);
+  });
+  return { on: onSpy };
+});
+
+jest.doMock('child_process', () => ({spawn}))
+
+import StartCmd from '../../src/commands/start'
+
+describe('Start command', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('runs next dev', async () => {
+    await StartCmd.run();
```

Comment:
Perhaps try passing in empty brackets here. I am guessing that might overwrite the argv array.